### ISSUE TITLE
pretrained densenet169 weights

### ISSUE THREAD

**Issue by Kexiii** [gpleiss/efficient_densenet_pytorch#40]
Hi, thanks for your great work!
I'm working on densenet169 these days, do you know where I can find the ImageNet pretrained weights for this efficient implementation? Or do you have any example code to show how to convert the other implementation's pretrained model to this one?
I do have noticed this #13 , but it seems @ZhengRui didn't provide any example code, and I don't know where to start..

**Comment by gpleiss** [comment]
I think you should be able to use any of the PyTorch pretrained DenseNets:

```
model_urls = {
    'densenet121': 'https://download.pytorch.org/models/densenet121-a639ec97.pth',
    'densenet169': 'https://download.pytorch.org/models/densenet169-b2777c0a.pth',
    'densenet201': 'https://download.pytorch.org/models/densenet201-c1103571.pth',
    'densenet161': 'https://download.pytorch.org/models/densenet161-8d451a50.pth',
}
```

Those urls should have the weights. Let me know if this works!

**Comment by gpleiss** [comment]
Did you try the ones here: https://github.com/mingminzhen/densenet-efficient-model

**Comment by Kexiii** [comment]
Yes, but it is based on the pretrained memory efficient torch model(232,264):
[https://github.com/liuzhuang13/DenseNet](url)
In the tech report, they didn't provide efficient densenet169 pretrained torch model that I want

**Comment by gpleiss** [comment]
Try converting the pytorch model then. All you have to do is change the name of the keys in the state duct in the link that I supplied. That’s how we created the other efficient densenets.

Best,
Geoff

**Comment by Kexiii** [comment]
Thanks for your advice,
want to let you know I've tried to convert the densenet169:[https://github.com/Kexiii/Pretrained-Efficient-DenseNet](https://github.com/Kexiii/Pretrained-Efficient-DenseNet)

**Comment by ZZXin** [comment]
> I think you should be able to use any of the PyTorch pretrained DenseNets:
> 
> ```
> model_urls = {
>     'densenet121': 'https://download.pytorch.org/models/densenet121-a639ec97.pth',
>     'densenet169': 'https://download.pytorch.org/models/densenet169-b2777c0a.pth',
>     'densenet201': 'https://download.pytorch.org/models/densenet201-c1103571.pth',
>     'densenet161': 'https://download.pytorch.org/models/densenet161-8d451a50.pth',
> }
> ```
> 
> Those urls should have the weights. Let me know if this works!

Hi,
I have a question On which dataset these weights are pretrained,ImageNet?

**Comment by gpleiss** [comment]
Yup. Imagenet.
